web/source/kmwrotation.ts

```diff
@@ -106,7 +106,7 @@ namespace com.keyman {
       } else if(os == 'Android') {
         // Android's far more consistent with its event generation than iOS.
         if('onmozorientationchange' in screen) {
-          util.attachDOMEvent(screen, 'mozorientationchange', function() {
+          util.attachDOMEvent(<any>screen, 'mozorientationchange', function() {
```

Comment:
Because:

```
Argument of type 'Screen' is not assignable to parameter of type 'EventTarget'.
  Type 'Screen' is missing the following properties from type 'EventTarget': addEventListener, dispatchEvent, removeEventListenerts(2345)
```

I guess this is due to `mozorientationchange` being a non-standard event.